tensorboard/webapp/widgets/resize_detector_test.ts

```diff
@@ -45,20 +45,20 @@ describe('resize detector', () => {
     }).compileComponents();
 
     resizeObservers = [];
-    spyOn(window, 'ResizeObserver').and.callFake(
-      (callback: ResizeObserverCallback) => {
-        const resizeObserver: ResizeObserver = {
-          disconnect: () => {},
-          observe: () => {},
-          unobserve: () => {},
-        };
-        triggerResize = () => {
-          callback([], resizeObserver);
-        };
-        resizeObservers.push(resizeObserver);
-        return resizeObserver;
-      }
-    );
+    spyOn(window, 'ResizeObserver').and.callFake(function (
+      callback: ResizeObserverCallback
+    ) {
```

Comment:
`function()` functions can be treated as constructors, while arrow functions cannot.  My hypothesis is that internally, some downleveling to ES5 converts the latter to the former, making it safe.  In OSS, the arrow function is preserved through to the test, causing it to fail.

Reply:
Hmm, perhaps there is a discrepancy in the typing shim?

Reply:
While there is a typing discrepancy, the error happens at runtime not build time.  I suppose it's possible our OSS type checking is less strict somehow.  Either that, or something internally rewrites the arrow fn into a function fn.